src/accountTransactions.ts

```diff
@@ -51,6 +57,77 @@ export class SimpleTransferWithMemoHandler
     }
 }
 
+export class DeployModuleHandler
+    implements AccountTransactionHandler<DeployModulePayload>
+{
+    getBaseEnergyCost(payload: DeployModulePayload): bigint {
+        const cost: number = Math.round((payload.content.length + 1) / 10);
+        return BigInt(cost);
+    }
+
+    serialize(transfer: DeployModulePayload): Buffer {
+        const serializedWasm = packBufferWithWord32Length(transfer.content);
+        const serializedVersion = encodeWord32(transfer.version);
+        return Buffer.concat([serializedVersion, serializedWasm]);
+    }
+}
+
+export class InitContractHandler
+    implements AccountTransactionHandler<InitContractPayload>
+{
+    getBaseEnergyCost(payload: InitContractPayload): bigint {
+        return payload.maxContractExecutionEnergy;
+    }
+
+    serialize(payload: InitContractPayload): Buffer {
+        const serializedAmount = encodeWord64(payload.amount.microGtuAmount);
+        const initNameBuffer = Buffer.from(payload.initName);
```

Comment:
Please provide the encoding explicitly to the `Buffer.from()` call. I suppose it is `UTF-8` here. This has to be done in all the places you use `Buffer.from()` on some text.